Production/plugins/PixelTrackTableProducer.cc

```diff
@@ -13,7 +13,8 @@
 #include "DataFormats/TrajectoryState/interface/LocalTrajectoryParameters.h"
```

Comment:
revert changes

Reply:
If we revert "#include "RecoPixelVertexing/PixelTrackFitting/interface/FitUtils.h" then the compilation will fail in CMSSW_13_3_X. Do you want to revert it?